client/asset/eth/eth.go

```diff
@@ -2756,17 +2756,45 @@ func (*baseWallet) ValidateSecret(secret, secretHash []byte) bool {
 }
 
 // SyncStatus is information about the blockchain sync status.
+//
+// TODO: Since the merge, the sync status from a geth full node, namely the
+// prog.CurrentBlock prog.HighestBlock, always seem to be the same number.
+// Initial sync will always be zero. Later when restarting the node they move
+// together but never indicate the highest know block on the chain. Further
+// more, requesting the best block header starts to fail after a few tries
+// during initial sync. Investigate how to get correct sync progress.
 func (eth *baseWallet) SyncStatus() (bool, float32, error) {
 	prog, err := eth.node.syncProgress(eth.ctx)
 	if err != nil {
 		return false, 0, err
 	}
+	checkHeaderTime := func() (bool, error) {
+		bh, err := eth.node.bestHeader(eth.ctx)
+		if err != nil {
+			return false, err
+		}
+		// Time in the header is in seconds.
+		timeDiff := time.Now().Unix() - int64(bh.Time)
+		if timeDiff > dexeth.MaxBlockInterval && eth.net != dex.Simnet {
+			eth.log.Debugf("Time since last block (%d sec) exceeds %d sec."+
+				"Assuming not in sync.", timeDiff, dexeth.MaxBlockInterval)
```

Comment:
Seems like our only option.  We may want to add a note to the message to check the system clock too, because that could wreck things.

Reply:
Added a note to the log and made it info level. Should it be a note in the ui? https://github.com/decred/dcrdex/compare/7e5a0222d0c2847c90137004201e11f777013285..a417333106afbfe8320818c2dc719bcd8aca6308

Reply:
> Should it be a note in the ui?

Don't think so.  They already get an implied "just wait longer" by it saying false for synced.  We could clamp it at 99.9% too though, if the best header is old, which we do for BTC as of recently.